jscomp/syntax/tests/conversion/reason/expected/fastPipe.res.txt

```diff
@@ -22,3 +22,16 @@ let aggregateTotal = (forecast, ~audienceType) =>
     views: item["reach"]["views"],
     sample: item["reach"]["sample"],
   })
+
+let myFunc = (strA, strB) => strA ++ strB
+
+"expr1"->myFunc("expr2")->myFunc("expr3")
```

Comment:
Why doesn't this break into several lines?

Reply:
Because it has  no newline in between the expressions